zerver/lib/actions.py

```diff
@@ -4882,9 +4882,9 @@ def user_info(um: UserMessage) -> Dict[str, Any]:
     if (len(changed_messages) > 0 and new_stream is not None and
             stream_being_edited is not None):
         # Notify users that the topic was moved.
-        notify_topic_moved_streams(user_profile, stream_being_edited, orig_topic_name,
-                                   new_stream, topic_name, send_notification_to_old_thread,
-                                   send_notification_to_new_thread)
+        send_message_moved_breadcrumbs(user_profile, stream_being_edited, orig_topic_name,
+                                       new_stream, topic_name, send_notification_to_old_thread,
+                                       send_notification_to_new_thread)
```

Comment:
This looks good but needs a rebase across the Black transition; I can probably get a copy from @abhijeetbodas2001's original PR as that looks to be rebased.

Reply:
OK, confirmed that in fact those commits are already merged, so you can drop this on rebase :).